firmware/target/VEGA/target.h

```diff
@@ -120,8 +113,6 @@ enum sens_type_e : uint32_t {
   IMU_ID_ACC,
```

Comment:
Let's modernize this enum as well:

 //TODO this can probably be reduced to uint8_t
 //either SensorType or SensorReadingType because it's an individual type of reading?
 enum class SensorType: uint32_t { 
  kInvalid
  kAcc
  kGyro
  kBaro
 }